ui/src/components/Search.js

```diff
@@ -145,14 +145,30 @@ class Search extends React.Component {
       return option.label;
     }
     if (option.facetValue !== null && option.facetValue !== "") {
-      return (
-        <div>
-          <span style={{ color: "#cccfd4" }}>Add</span>
-          <span> {option.facetName} </span>
-          <span style={{ color: "#cccfd4" }}>facet and select</span>
-          <span> {option.facetValue} </span>
-        </div>
-      );
+      if (option.isTimeSeries) {
```

Comment:
time series could have description

Reply:
The `else if (option.facetDescription != null)` statement is at a lower nesting level than the `if (option.isTimeSeries)` statement, so it should still apply to both time series and non time series fields, unless I'm missing a case?

Reply:
Ah ok